src/modules/Dashboard/features/RespondentDetails/RespondentDetails.styles.ts

```diff
@@ -0,0 +1,23 @@
+import { styled, Box } from '@mui/material';
+
+import { palette } from 'shared/styles/variables/palette';
+
+export const StyledTitleContainer = styled(Box)`
+  margin: 2rem 3rem;
```

Comment:
Let's use mui variables `theme.spacing...` for `margin/padding`  to bring the consistent spaces between component and maintain it easily.
For now `spacing = 10 (px)` https://github.com/ChildMindInstitute/mindlogger-admin/blob/main/src/shared/styles/theme.ts#16

Reply:
Great, thank you!